app/phpunit.xml.dist

```diff
@@ -27,6 +27,7 @@
     </testsuite>
     <testsuite name="Plugin tests">
       <directory>./../plugins/*Bundle/Tests</directory>
+      <exclude>./../plugins/MauticCrmBundle/Tests/Pipedrive</exclude>
```

Comment:
We run this test separately in CI now due to a bug. They do pass if they are ran separately. Since we're planning to decouple these plugins from core, I think it's OK for now to temporary have this situation.